datadog/dogstatsd/base.py

```diff
@@ -82,6 +83,14 @@ def __init__(self, host=DEFAULT_HOST, port=DEFAULT_PORT, max_buffer_size=50, nam
         If set, it is appended to the constant (global) tags of the statsd client.
         :type DD_VERSION: string
 
+        :envvar DD_TELEMETRY_HOST: the host for the dogstatsd server we wish to submit
+        telemetry stats to. If set, it overrides default value.
+        :type DD_TELEMETRY_HOST: string
+
+        :envvar DD_TELEMETRY_PORT: the port for the dogstatsd server we wish to submit
+        telemetry stats to. If set, it overrides default value.
+        :type DD_TELEMETRY_HOST: string
```

Comment:
Should be `DD_TELEMETRY_PORT` in place of `DD_TELEMETRY_HOST` here, no ?

Reply:
You're 100% right, thanks for pointing it out. We're bringing this PR back to life, hopefully we'll merge it soon.